content/id/docs/concepts/overview/working-with-objects/labels.md

```diff
@@ -0,0 +1,227 @@
+---
+reviewers:
+- mikedanese
+title: Label dan Selektor
```

Comment:
I've found "selector" in this glossarium from Pusat Bahasa http://bahasasastra.kemdikbud.go.id/glosarium/index.php?gloss_asing=selector&gloss_indonesia=&jenis=exact&Bidang=all&infocmd=Cari which is translated to "Selektor".

Besides, we also have other docs that using "Selektor" like this one https://kubernetes.io/id/docs/concepts/overview/working-with-objects/field-selectors/

Reply:
Sorry for the really belated comments. Kinda heavy workloads lately. 
Thinking about it, as long as it's comprehensible to the readers, it's okay to "invent" a word.
We might also focus on getting pages done instead of being blocked by these types of problems.
Let's just use Selektor then!